src/config/dynamic-theme-fixes.config

```diff
@@ -7075,12 +7075,20 @@ IGNORE IMAGE ANALYSIS
 ================================
 
 natemat.pl
+aszdziennik.pl
+innpoland.pl
+dadhero.pl
+mamadu.pl
 
 INVERT
-.header__brand
-.header__box__list--social--search
-.header__facebook
-.header__hamburger
+img[src*="/static/media/"]
+img[alt="close icon"]
+a[href="#comments"]
+
+CSS
+.css-1alc0q3-Footer--Logo {
```

Comment:
Hmm,
```
div?[class$="Footer--Logo"] {
```
Something along those lines sounds better.

Reply:
Code is ugly. 
![20210420-1618904111-001](https://user-images.githubusercontent.com/9846948/115356214-b86c6580-a1bb-11eb-8849-0dde8d49f49d.png)

Reply:
I see, but
```CSS
img[class*="Footer--Logo"] {
```
Should do it.